### PR TITLE
Todo list: Add/remove

### PR DESCRIPTION
**Pull request by Prisca-tech**
Remove all hardcoded items from the tasks array.
Create a new JavaScript file for the new functionality.
Implement a function for adding a new task
Implement a function for deleting a task
Implement a function for editing task descriptions.
By default new tasks should have the property completed set to false and the property index set to the value of the new array length
Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique
All changes to the To Do List should be saved in local storage.

### CODE REVIEW

File: modules/todo.js [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
+{
+  description: 'complete to-do list project',
+  completed: false,
+  index: 1,
+},
+{
+  description: 'go shopping',
+  completed: false,
+  index: 2,
+},
```
Comment (arslanbisharat) [inline modules/todo.js]: - [ ] please make sure the hard coded items have been removed from the list.

File: modules/addTodoList.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { todocontainer } from './variables.js';
+import deleteItem from './removeTodo.js';
+import edit from './edit.js';
+
```
Comment (arslanbisharat) [inline modules/addTodoList.js]: - [ ] By default, new tasks should have the property completed set to false and the property index set to the value of the new array length

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @Prisca-tech, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle:

Your project is complete. There is nothing else to say about your project. :+1:
Go ahead and merge it. :muscle:

## Highlights :1st_place_medal:
:heavy_check_mark: Following Github Flow
:heavy_check_mark: Following Linters
:heavy_check_mark: Add & Remove Buttons are working

### [Optional]

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! clap clap clap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**